[UA] Facilitate batch reindex command to user

sabarasaba opened the pull request:
## Summary
This PR adds support for bulk selecting ES deprecations and show a console command to reindex them.

#### How to test

* Follow the instructions in https://github.com/elastic/kibana-team/issues/1521. Use the data folder starting with [01] because it has the largest variety of deprecations.
* Start kibana with the changes of this pr checked out
* Verify that in UA -> ES Deprecations you can select multiple issues and if they are of the reindex type a CTA should appear that allows you to run the command in console.

<details>
<summary>Demo</summary>

https://github.com/user-attachments/assets/76828b36-27e5-4aaa-8304-caded64117ed


</details>

<details>
<summary>Screenshots</summary>

![Screenshot 2025-02-26 at 10 43 06](https://github.com/user-attachments/assets/3af83dca-d446-44d4-8db6-21b4410c6c85)

![Screenshot 2025-02-26 at 10 43 11](https://github.com/user-attachments/assets/d6c0cf22-6734-4050-86ba-968dd2c0270e)

![Screenshot 2025-02-26 at 10 43 15](https://github.com/user-attachments/assets/f755cc56-f7df-41fe-9550-d69fd7af621b)


</details>

sabarasaba committed:
Add multiple selection actions to es deprecation table

sabarasaba committed:
commit with @elastic email

sabarasaba committed:
Fix column size

sabarasaba committed:
Add reindexing modal and command display

sabarasaba committed:
Add open in console cta and show docs link

sabarasaba committed:
Fix linter issues

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/styled_components_mapping'

sabarasaba committed:
Refactor helper fns

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279090)
* Commit: 2a89b5d2070cfe5ad32d63347cfcc64744fcaaae

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/279090#01953d22-b552-43cc-9a35-56ca90bb5aae)

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279090","number":279090,"commit":"2a89b5d2070cfe5ad32d63347cfcc64744fcaaae"}],"number":279090}
buildkite-pr-comment-->

sabarasaba committed:
Fix types

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279099)
* Commit: 5066bcb21587565ab11fc1fb0fd1476585cf581b

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/279099#01953d46-172a-4611-b93f-13ad07d2ca79)

### History
* :broken_heart: [Build #279090](https://buildkite.com/elastic/kibana-pull-request/builds/279090) failed 2a89b5d2070cfe5ad32d63347cfcc64744fcaaae

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279099","number":279099,"commit":"5066bcb21587565ab11fc1fb0fd1476585cf581b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279090","number":279090,"commit":"2a89b5d2070cfe5ad32d63347cfcc64744fcaaae"}],"number":279099}
buildkite-pr-comment-->

sabarasaba committed:
Fix linter issue

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279112)
* Commit: aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/279112#01953d5f-5b97-40f5-9aa9-24c01d95da1c)

### History
* :broken_heart: [Build #279099](https://buildkite.com/elastic/kibana-pull-request/builds/279099) failed 5066bcb21587565ab11fc1fb0fd1476585cf581b
* :broken_heart: [Build #279090](https://buildkite.com/elastic/kibana-pull-request/builds/279090) failed 2a89b5d2070cfe5ad32d63347cfcc64744fcaaae

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279112","number":279112,"commit":"aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279099","number":279099,"commit":"5066bcb21587565ab11fc1fb0fd1476585cf581b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279090","number":279090,"commit":"2a89b5d2070cfe5ad32d63347cfcc64744fcaaae"}],"number":279112}
buildkite-pr-comment-->

sabarasaba committed:
CTA should be filled

florent-leborgne reviewed (commented):
No strong feeling on my comments. I figure we may want to add just a bit more context to the modal, maybe?

sabarasaba commented on x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/bulk_reindex/bulk_reindex_modal.tsx:
```diff
@@ -0,0 +1,137 @@
+              >
+                <FormattedMessage
+                  id="xpack.upgradeAssistant.esDeprecations.bulkReindexModal.openInConsoleButtonLabel"
+                  defaultMessage="Open in Console"
```
Oh this will just open console and populate the editor with the command without executing it, so I think I'll keep my version of the copy

sabarasaba committed:
Address copy review

sabarasaba commented:
Thanks for having a look @florent-leborgne! I've addressed your feedback with https://github.com/elastic/kibana/pull/212365/commits/8bfb87d521c78b24cda0505c2acb7cea985de808

sabarasaba committed:
Add tests

sabarasaba committed:
Add missing i18n

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279395)
* Commit: ddf9610ecb9e62fa0613bec236226bf631bcdda2

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/279395#019541f8-59d5-4e25-94fb-a8c5273160f7)

### History
* :broken_heart: [Build #279112](https://buildkite.com/elastic/kibana-pull-request/builds/279112) failed aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d
* :broken_heart: [Build #279099](https://buildkite.com/elastic/kibana-pull-request/builds/279099) failed 5066bcb21587565ab11fc1fb0fd1476585cf581b
* :broken_heart: [Build #279090](https://buildkite.com/elastic/kibana-pull-request/builds/279090) failed 2a89b5d2070cfe5ad32d63347cfcc64744fcaaae

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279395","number":279395,"commit":"ddf9610ecb9e62fa0613bec236226bf631bcdda2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279112","number":279112,"commit":"aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279099","number":279099,"commit":"5066bcb21587565ab11fc1fb0fd1476585cf581b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279090","number":279090,"commit":"2a89b5d2070cfe5ad32d63347cfcc64744fcaaae"}],"number":279395}
buildkite-pr-comment-->

sabarasaba committed:
Fix linter

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279416)
* Commit: 51e1a602229359bb23a53507f551f44513d7ec85

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/b6922a4ee559d25dcc0c3a962ade4b5327eb5116) | [after](https://github.com/elastic/kibana/commit/51e1a602229359bb23a53507f551f44513d7ec85) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 193 | 196 | +3 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/b6922a4ee559d25dcc0c3a962ade4b5327eb5116) | [after](https://github.com/elastic/kibana/commit/51e1a602229359bb23a53507f551f44513d7ec85) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 207.2KB | 215.4KB | +8.2KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/b6922a4ee559d25dcc0c3a962ade4b5327eb5116) | [after](https://github.com/elastic/kibana/commit/51e1a602229359bb23a53507f551f44513d7ec85) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 23.8KB | 23.8KB | +28.0B |



### History
* :broken_heart: [Build #279395](https://buildkite.com/elastic/kibana-pull-request/builds/279395) failed ddf9610ecb9e62fa0613bec236226bf631bcdda2
* :broken_heart: [Build #279112](https://buildkite.com/elastic/kibana-pull-request/builds/279112) failed aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d
* :broken_heart: [Build #279099](https://buildkite.com/elastic/kibana-pull-request/builds/279099) failed 5066bcb21587565ab11fc1fb0fd1476585cf581b
* :broken_heart: [Build #279090](https://buildkite.com/elastic/kibana-pull-request/builds/279090) failed 2a89b5d2070cfe5ad32d63347cfcc64744fcaaae

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279416","number":279416,"commit":"51e1a602229359bb23a53507f551f44513d7ec85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279395","number":279395,"commit":"ddf9610ecb9e62fa0613bec236226bf631bcdda2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279112","number":279112,"commit":"aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279099","number":279099,"commit":"5066bcb21587565ab11fc1fb0fd1476585cf581b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279090","number":279090,"commit":"2a89b5d2070cfe5ad32d63347cfcc64744fcaaae"}],"number":279416}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/kibana-management (Team:Kibana Management)

sabarasaba commented:
@elasticmachine merge upstream

elasticmachine committed:
Merge branch '8.x' into ua-bulk_reindex_command

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279469)
* Commit: cbd458178e7c70a1d006cad3689e5441a1a63abd

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/43671dcf33e380807ffba5bfbf9fce6d7ae40cf9) | [after](https://github.com/elastic/kibana/commit/cbd458178e7c70a1d006cad3689e5441a1a63abd) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 194 | 197 | +3 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/43671dcf33e380807ffba5bfbf9fce6d7ae40cf9) | [after](https://github.com/elastic/kibana/commit/cbd458178e7c70a1d006cad3689e5441a1a63abd) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 208.7KB | 216.9KB | +8.2KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/43671dcf33e380807ffba5bfbf9fce6d7ae40cf9) | [after](https://github.com/elastic/kibana/commit/cbd458178e7c70a1d006cad3689e5441a1a63abd) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 23.8KB | 23.8KB | +26.0B |



### History
* :green_heart: [Build #279416](https://buildkite.com/elastic/kibana-pull-request/builds/279416) succeeded 51e1a602229359bb23a53507f551f44513d7ec85
* :broken_heart: [Build #279395](https://buildkite.com/elastic/kibana-pull-request/builds/279395) failed ddf9610ecb9e62fa0613bec236226bf631bcdda2
* :broken_heart: [Build #279112](https://buildkite.com/elastic/kibana-pull-request/builds/279112) failed aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d
* :broken_heart: [Build #279099](https://buildkite.com/elastic/kibana-pull-request/builds/279099) failed 5066bcb21587565ab11fc1fb0fd1476585cf581b
* :broken_heart: [Build #279090](https://buildkite.com/elastic/kibana-pull-request/builds/279090) failed 2a89b5d2070cfe5ad32d63347cfcc64744fcaaae

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279469","number":279469,"commit":"cbd458178e7c70a1d006cad3689e5441a1a63abd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279416","number":279416,"commit":"51e1a602229359bb23a53507f551f44513d7ec85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279395","number":279395,"commit":"ddf9610ecb9e62fa0613bec236226bf631bcdda2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279112","number":279112,"commit":"aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279099","number":279099,"commit":"5066bcb21587565ab11fc1fb0fd1476585cf581b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279090","number":279090,"commit":"2a89b5d2070cfe5ad32d63347cfcc64744fcaaae"}],"number":279469}
buildkite-pr-comment-->

SoniaSanzV commented:
I don't think we should allow users to select indices that are already reindexed or in progress


https://github.com/user-attachments/assets/3383dc4e-41f7-4ede-a080-bec12ab44195

SoniaSanzV reviewed (commented):
Thank you for working on that @sabarasaba!! I've done some manual testing and I don't think we should allow users to select indices that are already reindexed or in progress, what do you think?

https://github.com/user-attachments/assets/3383dc4e-41f7-4ede-a080-bec12ab44195

 Apart from that, everything looks good to me. It has been so nice to reindex 34 indices with just 4 total clicks! 😻

sabarasaba committed:
Dont allow to reselect for reindexing if its already in progress or completed

sabarasaba commented:
> I've done some manual testing and I don't think we should allow users to select indices that are already reindexed or in progress, what do you think?

That's a very good and valid point, thanks for bringing that up! I've fixed that with https://github.com/elastic/kibana/pull/212365/commits/770829ba1da95bbb068504ab06513986556356e8


<details>
<summary>Screenshots</summary>

![Screenshot 2025-02-28 at 15 44 07](https://github.com/user-attachments/assets/6654a558-3f5f-4619-be29-7665ba655891)
![Screenshot 2025-02-28 at 15 43 49](https://github.com/user-attachments/assets/b2699053-d778-4434-829d-bfb44fe5a75d)


</details>

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/279997)
* Commit: 770829ba1da95bbb068504ab06513986556356e8

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/43671dcf33e380807ffba5bfbf9fce6d7ae40cf9) | [after](https://github.com/elastic/kibana/commit/770829ba1da95bbb068504ab06513986556356e8) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 194 | 197 | +3 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/43671dcf33e380807ffba5bfbf9fce6d7ae40cf9) | [after](https://github.com/elastic/kibana/commit/770829ba1da95bbb068504ab06513986556356e8) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 208.7KB | 217.0KB | +8.4KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/43671dcf33e380807ffba5bfbf9fce6d7ae40cf9) | [after](https://github.com/elastic/kibana/commit/770829ba1da95bbb068504ab06513986556356e8) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 23.8KB | 23.8KB | +26.0B |



### History
* :green_heart: [Build #279469](https://buildkite.com/elastic/kibana-pull-request/builds/279469) succeeded cbd458178e7c70a1d006cad3689e5441a1a63abd
* :green_heart: [Build #279416](https://buildkite.com/elastic/kibana-pull-request/builds/279416) succeeded 51e1a602229359bb23a53507f551f44513d7ec85
* :broken_heart: [Build #279395](https://buildkite.com/elastic/kibana-pull-request/builds/279395) failed ddf9610ecb9e62fa0613bec236226bf631bcdda2
* :broken_heart: [Build #279112](https://buildkite.com/elastic/kibana-pull-request/builds/279112) failed aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d
* :broken_heart: [Build #279099](https://buildkite.com/elastic/kibana-pull-request/builds/279099) failed 5066bcb21587565ab11fc1fb0fd1476585cf581b
* :broken_heart: [Build #279090](https://buildkite.com/elastic/kibana-pull-request/builds/279090) failed 2a89b5d2070cfe5ad32d63347cfcc64744fcaaae

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279997","number":279997,"commit":"770829ba1da95bbb068504ab06513986556356e8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279469","number":279469,"commit":"cbd458178e7c70a1d006cad3689e5441a1a63abd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279416","number":279416,"commit":"51e1a602229359bb23a53507f551f44513d7ec85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279395","number":279395,"commit":"ddf9610ecb9e62fa0613bec236226bf631bcdda2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279112","number":279112,"commit":"aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279099","number":279099,"commit":"5066bcb21587565ab11fc1fb0fd1476585cf581b"}],"number":279997}
buildkite-pr-comment-->

SoniaSanzV commented on x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/indices/resolution_table_cell.tsx:
```diff
@@ -100,6 +101,14 @@ const i18nTexts = {
   ),
 };
 
+export const canUserReindexRow = (reindexState: ReindexState) => {
```
Np: I find this constant name misleading, because if its value is true it cannot be reindexed. Maybe you can change the name (use `isInProgressOrCompleted` here instead of assigning this value to that variable later) or negate the whole block.

sabarasaba replied:
Yeah thats a good idea! https://github.com/elastic/kibana/pull/212365/commits/ac0ed6ed730879d20325fb8cf34ea219ec67ad9b

SoniaSanzV reviewed (approved):
I let you a nitpick comment, but I tested locally and it looks good to me :)

sabarasaba committed:
Rename fn

sabarasaba commented:
@elasticmachine merge upstream

elasticmachine committed:
Merge branch '8.x' into ua-bulk_reindex_command

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/280184)
* Commit: 76ba12581e6a6aaec598a56ee26ef7025f1d1dad

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/89b6339c357c7ec3ab0e7b83791be4cbf6990555) | [after](https://github.com/elastic/kibana/commit/76ba12581e6a6aaec598a56ee26ef7025f1d1dad) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 194 | 197 | +3 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/89b6339c357c7ec3ab0e7b83791be4cbf6990555) | [after](https://github.com/elastic/kibana/commit/76ba12581e6a6aaec598a56ee26ef7025f1d1dad) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 208.7KB | 217.1KB | +8.4KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/89b6339c357c7ec3ab0e7b83791be4cbf6990555) | [after](https://github.com/elastic/kibana/commit/76ba12581e6a6aaec598a56ee26ef7025f1d1dad) | diff |
| --- | --- | --- | --- |
| `upgradeAssistant` | 23.8KB | 23.8KB | +26.0B |



### History
* :green_heart: [Build #279997](https://buildkite.com/elastic/kibana-pull-request/builds/279997) succeeded 770829ba1da95bbb068504ab06513986556356e8
* :green_heart: [Build #279469](https://buildkite.com/elastic/kibana-pull-request/builds/279469) succeeded cbd458178e7c70a1d006cad3689e5441a1a63abd
* :green_heart: [Build #279416](https://buildkite.com/elastic/kibana-pull-request/builds/279416) succeeded 51e1a602229359bb23a53507f551f44513d7ec85
* :broken_heart: [Build #279395](https://buildkite.com/elastic/kibana-pull-request/builds/279395) failed ddf9610ecb9e62fa0613bec236226bf631bcdda2
* :broken_heart: [Build #279112](https://buildkite.com/elastic/kibana-pull-request/builds/279112) failed aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d
* :broken_heart: [Build #279099](https://buildkite.com/elastic/kibana-pull-request/builds/279099) failed 5066bcb21587565ab11fc1fb0fd1476585cf581b

cc @sabarasaba

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/280184","number":280184,"commit":"76ba12581e6a6aaec598a56ee26ef7025f1d1dad"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279997","number":279997,"commit":"770829ba1da95bbb068504ab06513986556356e8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279469","number":279469,"commit":"cbd458178e7c70a1d006cad3689e5441a1a63abd"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279416","number":279416,"commit":"51e1a602229359bb23a53507f551f44513d7ec85"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279395","number":279395,"commit":"ddf9610ecb9e62fa0613bec236226bf631bcdda2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/279112","number":279112,"commit":"aa90fee1b61413dc6c4d48a30fd6723cd0e7a92d"}],"number":280184}
buildkite-pr-comment-->

sabarasaba commented:
Closing for now as its unlikely we'll get this merged
